Documentation/MAX78000.html

```diff
@@ -63,53 +63,72 @@ <h2>Example Applications</h2>
+				<li><a href='../Examples/MAX78000/CNN/cifar-10/README.md'>cifar-10</a> - Object Classification using 10 Choices</li>
+				<li><a href='../Examples/MAX78000/CNN/cifar-100-mixed/README.md'>cifar-100-mixed</a> - Object Classification using 100 Choices</li>
+				<li><a href='../Examples/MAX78000/CNN/cifar-100-residual/README.md'>cifar-100-residual</a> - Object Classification using 100 Choices</li>
+				<li><a href='../Examples/MAX78000/CNN/cifar-100-simplewide2x-mixed/README.md'>cifar-100-simplewide2x-mixed</a> - Object Classification using 100 Choices</li>
 				<li><a href='../Examples/MAX78000/CNN/cifar-100/README.md'>cifar-100</a> - Object Classification using 100 Choices</li>
+				<li><a href='../Examples/MAX78000/CNN/digit-detection-demo/README.md'>digit-detection-demo</a> - Numerical Digit Detection</li>
 				<li><a href='../Examples/MAX78000/CNN/faceid/README.md'>faceid</a> - Facial Recognition Demonstration</li>
 				<li><a href='../Examples/MAX78000/CNN/faceid_demo/README.md'>faceid_demo</a> - Facial Recognition Demonstration</li>
-				<li><a href='../Examples/MAX78000/CNN/faceid_evkit/README.md'>faceid_evkit</a> - Facial Recognition Demonstration</li>
 				<li><a href='../Examples/MAX78000/CNN/faceid_evkit-riscv/README.md'>faceid_evkit</a> - Facial Recognition Demonstration (runs on RISC-V core)</li>
-				<li><a href='../Examples/MAX78000/CNN/kws20_demo/README.md'>kws20_demo</a> - Audio Keyword Spotting</li>
+				<li><a href='../Examples/MAX78000/CNN/faceid_evkit/README.md'>faceid_evkit</a> - Facial Recognition Demonstration</li>
 				<li><a href='../Examples/MAX78000/CNN/kws20_demo-riscv/README.md'>kws20_demo-riscv</a> - Audio Keyword Spotting (runs on RISC-V core)</li>
+				<li><a href='../Examples/MAX78000/CNN/kws20_demo/README.md'>kws20_demo</a> - Audio Keyword Spotting</li>
 				<li><a href='../Examples/MAX78000/CNN/kws20_v3/README.md'>kws20_v3</a> - Audio Keyword Spotting</li>
-				<li><a href='../Examples/MAX78000/CNN/mnist/README.md'>mnist</a> - Handwritten Digit Recognition</li>
 				<li><a href='../Examples/MAX78000/CNN/mnist-riscv/README.md'>mnist-riscv</a> - Handwritten Digit Recognition (runs on RISC-V core)</li>
 				<li><a href='../Examples/MAX78000/CNN/mnist-streaming/README.md'>mnist-streaming</a> - Handwritten Digit Recognition</li>
+				<li><a href='../Examples/MAX78000/CNN/mnist/README.md'>mnist</a> - Handwritten Digit Recognition</li>
+				<li><a href='../Examples/MAX78000/CNN/rps-demo/README.md'>rps-demo</a> - Rock, Paper, Scissors Demo</li>
 				<li><a href='../Examples/MAX78000/CNN/rps/README.md'>rps</a> - Rock, Paper, Scissors Hand Gesture Recognition</li>
-				<li><a href='../Examples/MAX78000/CNN/rps_demo/README.md'>rps_demo</a> - Rock, Paper, Scissors Demo</li>
 				<li><a href='../Examples/MAX78000/CNN/snake_game_demo/README.md'>snake_game_demo</a> - Audio Keyword Spotting</li>
-				<li><a href='../Examples/MAX78000/CNN/UNet-demo/README.md'>UNet-demo</a> - U-Net Network Demo</li>
```

Comment:
Are we waiting on the README for CNN/svhn_tinierssd before including that example in the list?

Reply:
Yes.